pkg/core/netmap/prettyprint.go

```diff
@@ -1,4 +1,4 @@
-package common
+package netmap
```

Comment:
could you, please, make that move in a separate commit?

Reply:
In this case I need to add in `adm` dependency on `cli` or duplicate code. Is It ok?

Reply:
Updated.

Reply:
Moved from `pkg` to `cmd/internal/common`